modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ImagePicker/ImagePicker.es.js

```diff
@@ -276,6 +280,15 @@ const Main = ({
 	value,
 	...otherProps
 }) => {
+	const prevEditingLanguageId = usePrevious(editingLanguageId);
+
+	if (prevEditingLanguageId !== editingLanguageId && localizable) {
+		value =
+			localizedValue[editingLanguageId] !== undefined
+				? localizedValue[editingLanguageId]
+				: localizedValue[defaultLanguageId];
+	}
```

Comment:
Hi @MirnaGama, good job! 
Just one suggestion to simplify the code, we can use the _Nullish Coalescing_ operator in this case.

```suggestion
	if (prevEditingLanguageId !== editingLanguageId && localizable) {
		value = localizedValue[editingLanguageId] ?? localizedValue[defaultLanguageId];
	}
```

Thanks!

Reply:
@alinedoleron Done ! :grin: